subjects/mobile-dev/stock-market/resources/mock-stock-data-server/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, jsonify
+from flask_cors import CORS
+
+import time
+from utils import load_historical_data 
+
+app = Flask(__name__)
+CORS(app)
+
+start_time = time.time()
+
+historical_data = load_historical_data()  # Dictionary to store historical data
+
+@app.route('/exchange_rate/<symbol>')
+def get_stock_data(symbol):
+    if symbol not in list(historical_data.keys()):
+        return jsonify("Invalid symbol")
+    current_time = time.time()
+    step = (int(current_time * 10) - int(start_time * 10)) % len(historical_data[symbol])
+    try:
+        return jsonify({
+            'symbol': 'USD',
```

Comment:
Hardcoded symbol !!
shoudn't it be the parameter symbol instead of 'USD' ?

Reply:
I have added this information to make the JSON results "shape" looks more alike a real API result. This information is hardcoded since it is not present in the CSV files, the source of the data. If you have a more elegant way to include this info or a strong point to remove it, please, let me know - if not I would keep it as it is